src/main/java/com/minecolonies/blocks/BlockSubstitution.java

```diff
@@ -61,6 +61,8 @@ private void initBlock()
      *
      * @return true
      */
+    //todo: remove once we no longer need to support this
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1135)